tedana/workflows/tedana.py

```diff
@@ -302,20 +302,25 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
         n_components, dd = decomposition.tedpca(catd, data_oc, combmode, mask,
                                                 t2s, t2sG, ref_img,
                                                 tes=tes, method=tedpca, ste=ste,
-                                                kdaw=10., rdaw=1., wvpca=wvpca)
+                                                kdaw=10., rdaw=1., wvpca=wvpca,
+                                                verbose=verbose)
         mmix_orig, fixed_seed = decomposition.tedica(n_components, dd,
                                                      fixed_seed)
 
         if verbose:
             np.savetxt(op.join(out_dir, '__meica_mix.1D'), mmix_orig)
+            if ste == -1:
+                io.filewrite(utils.unmask(dd, mask),
+                             op.join(out_dir, 'tsoc_whitened.nii'), ref_img)
```

Comment:
Can we make this consistent with the `tsoc` filename (i.e., either `ts_OC_whitened.nii` or `tsoc.nii`) ?

Reply:
Can do.